add-on/src/lib/dir-view.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const filesize = require('filesize')
-const mainStyle = require('ipfs/src/http/gateway/dir-view/style')
+const mainStyle = require('ipfs-http-response/src/dir-view/style')
```

Comment:
Does this mean we can remove `ipfs/src/http/gateway/dir-view/` from companion repo?

Reply:
No, that's a path in the ipfs module